html/semantics/embedded-content/the-object-element/usemap-casing.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>object usemap case-sensitive</title>
+<link rel="author" title="Domenic Denicola" href="mailto:d@domenic.me">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/infrastructure.html#rules-for-parsing-a-hash-name-reference">
+<!-- See also: https://github.com/whatwg/html/issues/1666 -->
+
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<div id="log"></div>
+
+<object data="/images/threecolors.png" usemap="#sImPlE" width="300" height="300"></object>
```

Comment:
Interesting. Some, but not all, browsers seem to fail such a sanity check for object elements. I also am getting different results between the wpt-stability-bot (presumably Linux) and my Windows 10 computer.

I guess it'll be a good test to add regardless.